contracts/interchain-token-service/src/contract.rs

```diff
@@ -55,9 +56,7 @@ pub enum Error {
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: Empty) -> Result<Response, ContractError> {
-    // Implement migration logic if needed
-
-    // TODO migrate max uint to max uint bits
+    migrations::v1_1_0::migrate(deps.storage)?;
```

Comment:
could you use the new format for migrations. The router is already using that format, basically move the `migrate` endpoint and everything related to the migration to the mod and re-export as necessary

Reply:
note the use of the macro `migrate_from_version`

Reply:
convention checklist failure lol